filter/filter.go

```diff
@@ -180,6 +182,11 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			// https://github.com/sahilm/fuzzy
 			m.matches = fuzzy.Find(m.textinput.Value(), m.choices)
 
+			// For exact search return only the exact matches
+			if m.exact {
+				m.matches = exactMatches(m.textinput.Value(), m.matches)
+			}
```

Comment:
I think we can put the fuzzy finding in the else block here as it's either exact or fuzzy.

Reply:
Ah, yes, right! We can filter directly the `choices`. And since they are stable I've also removed the sort. 👍